pages/review-barrier.tsx

```diff
@@ -0,0 +1,365 @@
+                      },
+                      { bold },
+                    )}
+                  </h4>
+                  <p
+                    data-h2-padding="b(all, s)"
+                    data-h2-margin="b(bottom, none)"
+                    data-h2-bg-color="b(lightgray)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "You haven't marked any solutions as working for this particular barrier.",
+                    })}
+                  </p>
+                  <h4
+                    data-h2-font-size="b(h3)"
+                    data-h2-margin="b(bottom, none)"
+                  >
+                    {intl.formatMessage(
+                      {
+                        defaultMessage: "Attempted Solutions",
+                      },
+                      { bold },
+                    )}
+                  </h4>
+                  <p
+                    data-h2-padding="b(all, s)"
+                    data-h2-margin="b(bottom, none)"
+                    data-h2-bg-color="b(lightgray)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "You haven't marked any solutions as ineffective for this particular barrier.",
+                    })}
+                  </p>
+                </div>
+              </LeftSection>
+              <RightSection>
+                <h4 data-h2-font-size="b(h3)" data-h2-margin="b(bottom, none)">
+                  {intl.formatMessage(
+                    {
+                      defaultMessage: "Editing tools",
+                    },
+                    { bold },
+                  )}
+                </h4>
+                <p data-h2-margin="b(bottom, m)">
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Select the tools below to edit the barrier/solution information that appears on this page or identify a new solution for this barrier.",
+                  })}
+                </p>
+                <div data-h2-margin="b(bottom, m)">
+                  <Button
+                    type="submit"
+                    color="blue"
+                    mode="solid"
+                    data-h2-font-style="b(underline)"
+                    data-h2-padding="b(all, s)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Edit this barrier's information",
+                    })}
+                  </Button>
+                </div>
+                <div>
+                  <Button
+                    type="submit"
+                    color="blue"
+                    mode="solid"
+                    data-h2-font-style="b(underline)"
+                    data-h2-padding="b(all, s)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Propose a new solution",
```

Comment:
This should be changed as well.
![image](https://user-images.githubusercontent.com/22059495/165823344-3f95b337-3c81-4058-a35b-53fd252eef41.png)